### PR TITLE
Update seo.md

### PR DESCRIPTION
**Pull request by mtoribi**
## Overview

> Updated URL standards

### CODE REVIEW

File: testing/nonfunctional/seo.md [hunk 1]
```diff
@@ -68,14 +68,15 @@ More info from [Google Support](https://support.google.com/webmasters/answer/356
 
 URLs should be consistent, human-readable, and include targeted keywords. URLs are very important Search Relevance signal.
 
+- [ ] base URL is the main URL without product variations or facets
```
Comment (githubjosh) [inline testing/nonfunctional/seo.md]: @mtoribi 
I'd suggest moving this to the bottom of the list and updating:
"base url does not include variants (facets / dimensions), query strings, or fragments"

File: testing/nonfunctional/seo.md [hunk 2]
```diff
@@ -68,14 +68,15 @@ More info from [Google Support](https://support.google.com/webmasters/answer/356
...
 - [ ] each word in the URL phrase should be separated by a dash (`-`). 
 - [ ] URL phrase must include one or more of the comma-separated values found in the meta keywords element.
-- [ ] URLs should be short and easy to remember. Practicing this will prevent the need for additional vanity URLs.
+- [ ] URLs should be short and easy to remember with a URL length of 90 characters or less
```
Comment (githubjosh) [inline testing/nonfunctional/seo.md]: @mtoribi 
I'd suggest changing to "Maximum 90 character length - URLs should be short and easy to remember"
i.e. state rule followed by rationale

File: testing/nonfunctional/seo.md [hunk 3]
```diff
@@ -68,14 +68,15 @@ More info from [Google Support](https://support.google.com/webmasters/answer/356
...
 URLs should be consistent, human-readable, and include targeted keywords. URLs are very important Search Relevance signal.
 
+- [ ] base URL is the main URL without product variations or facets
 - [ ] each word in the URL phrase should be separated by a dash (`-`). 
 - [ ] URL phrase must include one or more of the comma-separated values found in the meta keywords element.
-- [ ] URLs should be short and easy to remember. Practicing this will prevent the need for additional vanity URLs.
+- [ ] URLs should be short and easy to remember with a URL length of 90 characters or less 
+- [ ] Max of 3 paths excluding language and region parameters
 
 For example, for the following `<meta name="keywords" content="smartphone, Apple iPhone X, wireless charging">`
```
Comment (githubjosh) [inline testing/nonfunctional/seo.md]: @mtoribi 
Might suggest changing "smartphone" to "phones" to better illustrate keyword use in URL

**Comment by ahmadnassri** [comment]
should we close this PR in lieu of the one just merged? @mtoribi

**Comment by mtoribi** [comment]
> should we close this PR in lieu of the one just merged? @mtoribi

Yes, please. @ahmadnassri . Can you help me with that? Not sure how to do it.

**Comment by githubjosh** [comment]
merged in another PR - https://github.com/telus/reference-architecture/pull/182

**Pull request closed without merging**